colossalai/inference/core/engine.py

```diff
@@ -184,13 +189,14 @@ def add_request(
                 prompt = None
             else:
                 prompt = prompts[i]
+            block_table = torch.full([self.inference_config.max_seq_len], -1, device=self.device)
```

Comment:
should be `max_block_num`

Reply:
We don't have a parameter called max_block_num.

Reply:
https://github.com/hpcaitech/ColossalAI/blob/eb672648c89e7f0e47cb6628c7c9f044bd1137ad/colossalai/inference/kv_cache/kvcache_manager.py#L118C34-L118C34